src/util/file_util.cpp

```diff
@@ -255,6 +255,11 @@ std::string fileutl_absolute_path(std::string const &path)
   return std::string(&(buffer[0]));
 #else
   char *absolute = realpath(path.c_str(), nullptr);
```

Comment:
I think you underestimate how long it would take to lint all the existing code created for the security project. I suggest speaking to Reuben about clang.